src/pages/GuestProfilePage/components/GuestProfile/GuestProfileInfo/GuestProfileInfo.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC, useEffect, useState } from "react";
```

Comment:
Try to use the appropriate structure of imports. Failures are happening during the build process. Always build a project before pushing commits, please